howto/stream/exchange-oob-data.md

```diff
@@ -269,6 +270,50 @@ try {
     ex.printStackTrace();
 }
 ```
+
+#### Install the APK as system app
+
+To connect the data channel to the Anbox WebRTC data proxy service within an Android container, the Android app must be installed and running as a system app. To do so, proceed with the following steps:
+1. Add the attribute `android:sharedUserId="android.uid.system"` into the `<manifest>` tag in the AndroidManifest.xml file of your Android app
+2. Create an addon to install your APK as a system app through the pre-start hook with the following content:
```

Comment:
Maybe link to Create an addon tutorial as well: https://documentation.ubuntu.com/anbox-cloud/en/latest/tutorial/creating-addon/

Reply:
This is not a deal breaker for me, if you feel the given steps are self-sufficient for creating the addon for this case, let's keep it as is.

Reply:
Let's keep it as is then.